deploy/helm/kuberhealthy/templates/configmap.yaml

```diff
@@ -28,6 +28,14 @@ data:
           severity: critical
         annotations:
           description: Kuberhealthy shows that the cluster is not healthy
+      - alert: DeadMansSwitch
+        expr: vector(1)
+        labels:
+          severity: critical
+        annotations:
+          description: This is a DeadMansSwitch meant to ensure that the entire Alerting
+          pipeline is functional.
+        summary: Alerting DeadMansSwitch
```

Comment:
curious what this is for?

Reply:
these must be coming from merging...although I thought this was only master that I had been doing it too....I am not sure but will get rid of it